data/sql/updates/pending_db_world/myfirstsql.sql

```diff
@@ -0,0 +1,14 @@
+--
+INSERT INTO `acore_world`.`creature_text` (`CreatureID`, `Text`, `Type`, `Probability`, `BroadcastTextId`) VALUES ('15111', 'I gonna make you into mojo!', '12', '100', '10435');
+INSERT INTO `acore_world`.`creature_text` (`CreatureID`, `ID`, `Text`, `Type`, `Probability`, `BroadcastTextId`) VALUES ('15111', '1', 'Troll mojo da strongest mojo!', '12', '100', '10437');
```

Comment:
```suggestion
INSERT INTO `creature_text` (`CreatureID`, `Text`, `Type`, `Probability`, `BroadcastTextId`) VALUES ('15111', 'I gonna make you into mojo!', '12', '100', '10435');
INSERT INTO `creature_text` (`CreatureID`, `ID`, `Text`, `Type`, `Probability`, `BroadcastTextId`) VALUES ('15111', '1', 'Troll mojo da strongest mojo!', '12', '100', '10437');
```

Reply:
Makes sense. I should've noticed that Keira and Heidi had different lines. Thanks!